controller_manager/src/ros2_control_node.cpp

```diff
@@ -41,17 +41,19 @@ int main(int argc, char ** argv)
     RCLCPP_INFO(cm->get_logger(), "update rate is %d Hz", cm->get_update_rate());
 
     rclcpp::Time begin = cm->now();
+    rclcpp::Time begin_last(begin);
+    rclcpp::Time end(begin);
```

Comment:
Some nit-picking on a nitpick :smile: 

```suggestion
    rclcpp::Time begin_last;
    rclcpp::Time end;
```

Reply:
I think we should initialize them, though. I could use this instead, if you prefer:

```
rclcpp::Time begin_last = begin;
rclcpp::Time end = begin;
```